components/BlogListing/BlogListing.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import { ISbStoryData, SbBlokData, getStoryblokApi } from '@storyblok/react';
+import styles from './BlogListing.module.scss';
+import Link from 'next/link';
+
+interface StoriesData {
+  stories: ISbStoryData[];
+}
+
+const BlogListing = () => {
+  const [data, setData] = useState<StoriesData | null>(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const storyblokApi = getStoryblokApi();
+        const response = await storyblokApi.getStories({
+          content_type: 'blogPost',
+        });
+        setData(response.data);
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  if (!data || !data.stories) {
+    console.error('No blog posts found');
+    return null;
+  }
+
+  console.log(data);
+
+  return (
+    <div className='mx-20 flex flex-wrap justify-center'>
+      {data.stories
+        .slice()
```

Comment:
@Sangram-Appcore I think we don't need to do slice() and reverse()
if we need to sort Stories, we should be sorting them via API (getStories supports sort order)